extensions/banner/runtime/src/main/resources/default_banner.txt

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
I do wish we had a better default banner.

Reply:
I will try to add different one.

Reply:
I have added this one as suggested by @gastaldi with slight modifications